package.json

```diff
@@ -1,8 +1,8 @@
 {
   "name": "@devexperts/remote-data-ts",
   "version": "0.3.1",
-  "main": "dist/remote-data.js",
```

Comment:
breaking change - just a note

Reply:
You mean when someone reference the file directly?